quantinuum-hugr/src/types.rs

```diff
@@ -42,6 +43,8 @@ pub enum EdgeKind {
     Value(Type),
     /// A reference to a static value definition.
     Static(Type),
+    /// A (perhaps-polymorphic) function (TLD)
+    Function(PolyFuncType),
```

Comment:
Renaming Static+Function to StaticValue and StaticFunction would be a possibility, or renaming Static edges to Const (their only sources) - they were once called ConstE edges...